pkg/virt-handler/vm.go

```diff
@@ -2802,10 +2804,14 @@ func isACPIEnabled(vmi *v1.VirtualMachineInstance, domain *api.Domain) bool {
 
 func (d *VirtualMachineController) isHostModelMigratable(vmi *v1.VirtualMachineInstance) error {
 	if cpu := vmi.Spec.Domain.CPU; cpu != nil && cpu.Model == v1.CPUModeHostModel {
-		node, err := d.clientset.CoreV1().Nodes().Get(context.Background(), vmi.Status.NodeName, metav1.GetOptions{})
+		obj, exists, err := d.virtHandlerNodeInformer.GetStore().GetByKey(vmi.Status.NodeName)
```

Comment:
This one definitely has to be fixed! :+1:

Reply:
@davidvossel @rthallisey @marceloamaral so here we have the reason for the high get on the nodes which we discussed in the perf meeting :)

Reply:
Sorry for all the back and forth, but I finally thought more about this (and realized that we add an informer just now). It looks to me, like all we need is understanding if the node-labeller could detect the host model. This info is already inside the virt-handler binary (since node-labeller is running there). We can directly look up this info from within the virt-handler binary and don't have to introduce a new watcher (on big clusters creating 500 additinal watchers, only watching one node each, but still).